<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeasurementBoundDialog</class>
 <widget class="QDialog" name="MeasurementBoundDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>371</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item alignment="Qt::AlignTop">
     <widget class="QLabel" name="name_label">
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="name"/>
    </item>
    <item>
     <widget class="QLabel" name="lower_bound_label">
      <property name="text">
       <string>Lower bound</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lower_bound"/>
    </item>
    <item>
     <widget class="QCheckBox" name="inclusive_lb">
      <property name="text">
       <string>Inclusive lower bound</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="upper_bound_label">
      <property name="text">
       <string>Upper bound</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="upper_bound"/>
    </item>
    <item>
     <widget class="QCheckBox" name="inclusive_ub">
      <property name="text">
       <string>Inclusive upper bound</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>name</tabstop>
  <tabstop>lower_bound</tabstop>
  <tabstop>inclusive_lb</tabstop>
  <tabstop>upper_bound</tabstop>
  <tabstop>inclusive_ub</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MeasurementBoundDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MeasurementBoundDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
